src/ServiceStack.EventStore/Exceptions/EventNotFoundException.cs

```diff
@@ -5,6 +5,7 @@ namespace ServiceStack.EventStore.Exceptions
 {
     using System;
 
+    [Serializable]
     public class EventNotFoundException: Exception
     {
         private readonly string stream;
```

Comment:
That makes sense. I have renamed it StreamId and changed it `public readonly` since otherwise it's a pointless variable.